projects/frontend-ui/shared-components/projects/gui/projects/shared/README.md

```diff
@@ -0,0 +1,26 @@
+# VDK Shared Components library
+
+This project implements common UI functionality that is used in the data-pipelines project.
+But these components are also used by private projects within VMWare.
```

Comment:
this is duplicated with the other readme / redundant?

Reply:
Removed. 
This duplication kinda existed in the source repo.